decidim-core/app/views/decidim/endorsements/_identity.html.erb

```diff
@@ -1,4 +1,4 @@
-<button type="button" <%= selected ? 'class="is-selected"'.html_safe : "" %> data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
+<button type="button" class="<%= "is-selected" if selected %>" data-user-identity data-method="<%= http_method.to_s.upcase %>" data-create_url="<%= create_url %>" data-destroy_url="<%= destroy_url %>">
```

Comment:
Reverted while based on the investigation performed.

Reply:
Great!